partner_changeset/security/security.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
```

Comment:
Not sure about this one. Is it possible with `<odoo>` to have a part of the records with noupdate and a part without?